### ISSUE TITLE
Oak bug

### ISSUE THREAD

**Issue by RealYezper** [bensku/Skript#1178]
So this is my script
on break of oak leaves:
	if player is not a op:
		cancel event
on break of spruce leaves:
	drop 1 light blue dye named "&9&lFlax"
	wait 64 seconds
	set block at event-block to spruce leaves

for some reason i cant destroy the spruce leave.. its like it thinks its oak leaves.. and oak wood is behaving the same way. type "on break of oak planks:" then i cant destroy ANY planks...

**Comment by Blueyescat** [comment]
I don't think that can be possible, at least with the up to date Skript and aliases. 

Get Skript from: https://github.com/bensku/Skript/releases

Replace your **`Skript/aliases-english.sk`** file with: [aliases-english.sk](https://raw.githubusercontent.com/Blueyescat/skAliases/b9b8f80b8af2ec5099d4816e4dfefa21930b2252/aliases-english.sk) *([PR](https://github.com/tim740/skAliases/pull/9) to [tim740/skAliases](https://github.com/tim740))*

**Comment by RealYezper** [comment]
it still happens.. i changed the code a little bit
on break of oak leaves:
	if player is not a op:
		cancel event

on break of spruce leaves:
	drop 1 light blue dye named "&9&lFlax"
	wait 30 seconds
	set block at event-block to oak planks

the spruce leaves is coming back after 1 second (cancel event from oak leaves) and after 30 seconds  its changing to oak planks.. O.o

**Comment by RealYezper** [comment]
okay i fixed it i changed it again to:
on break of spruce leaves:
	drop 1 light blue dye named "&9&lFlax"
	set block at event-block to air
	wait 30 seconds
	set block at event-block to oak planks
and now it works fine... 
BUT Thanks for your reply! :D
